libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Common/WebViewStateManager.swift

```diff
@@ -23,7 +23,8 @@
 import Foundation
 import WebKit
 
-public class SFSDKWebViewStateManager: NSObject {
+@objc(SFSDKWebViewStateManager)
+public class WebViewStateManager: NSObject {
```

Comment:
From [this thread](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/pull/3847#discussion_r2017517228), if we want to have a better Swift name we should have a deprecation path for the old one